src/App.jsx

```diff
@@ -50,14 +90,28 @@ function App() {
   return (
     <div className="app">
       <header>
-        <h1>4% Retirement Rule Calculator</h1>
-        <p>Calculate how much you can safely withdraw in retirement using the 4% rule</p>
+        <h1>Retirement Withdrawal Calculator</h1>
+        <p>Calculate how much you can safely withdraw in retirement using different strategies</p>
       </header>
 
       <div className="calculator">
         <div className="input-section">
           <h2>Your Financial Information</h2>
           
+          <div className="input-group">
+            <label htmlFor="withdrawalStrategy">Withdrawal Strategy</label>
+            <select
+              id="withdrawalStrategy"
```

Comment:
🔰 **Junior Developer**

Wow, I love the new strategy selector dropdown in `src/App.jsx` (lines 90-104)!! 🎉✨ Dropdowns make everything more interactive, and I saw a tweet yesterday about how dropdowns are basically the future of UX! Actually, I think we should refactor this into a React component called `StrategyDropdown`, and maybe put it in its own file for super clean code separation. Also, have you considered making the strategies dynamic from an API so we can add new ones without redeploying? Maybe TypeScript enums for the strategy keys? Just an idea! Hope this helps! 😊